library/date/today_spec.rb

```diff
@@ -2,5 +2,16 @@
 require 'date'
 
 describe "Date.today" do
-  it "needs to be reviewed for spec completeness"
+  it "returns a Date object" do
+    Date.today.should be_kind_of Date
+  end
+
+  it "sets Date object to the current date" do
+    today = Date.today
+    
+    now = Time.now
```

Comment:
This has the same problem, the day could change between those two calls, and below the seconds could change.
I am thinking:
```ruby
date = Date.today
now = Time.now
date_as_time = Time.new(today.year, today.mon, today.day) # Or maybe simply date.to_time
(date_as_time - now).should be_close(24 * 60 * 60)
```

Reply:
This threw me off, because be_close needs two parameters, not one.  be_close(0.0, 24 * 60 * 60) works, and I agree calling to_time makes sense because it saves us a variable and it's drops the relevance of the conversion in our assertion.  Pushing correction now.

Reply:
I am sorry, I was working from memory.